webapp/install_mattermost_webapp.sh

```diff
@@ -0,0 +1,27 @@
+readonly COMMITHASH=`jq -r '.localPackages.mattermost_webapp' package.json`
+
+echo "\n\nInstalling mattermost-webapp from the mattermost repo, using commit hash $COMMITHASH\n"
+
+if [ ! -d mattermost-webapp ]; then
+  mkdir mattermost-webapp
+fi
+
+cd mattermost-webapp
+
+if [ ! -d .git ]; then
+  git init
+  git config --local uploadpack.allowReachableSHA1InWant true
+  git remote add origin https://github.com/mattermost/mattermost.git
+fi
+
+git fetch --depth=1 origin $COMMITHASH
+git reset --hard FETCH_HEAD
+
+cd ..
+mv mattermost-webapp/webapp .
+rm -rf mattermost-webapp
+mv webapp mattermost-webapp
+
+npm i --save-dev ./mattermost-webapp/channels
+npm i --save-dev ./mattermost-webapp/platform/types
+npm i --save-dev ./mattermost-webapp/platform/client
```

Comment:
Just to be sure, do these still need to be `npm install`ed when they're already listed as `file:` dependencies in the `package.json`?

Reply:
I tried it without the npm install part but it didn't work.

Reply:
Thanks for confirming that. I would've hoped that NPM could still install dependencies that way correctly anyway, but I'm not surprised that this requires something a bit unexpected given this whole thing is fairly non-standard